src/main/java/appeng/me/helpers/MEMonitorHandler.java

```diff
@@ -42,11 +46,9 @@
  * listeners.
  *
  * @param <T>
- * @deprecated
  *
- * TODO: Needs to be redesigned to solve performance issues. Also should not be part of the API as class.
+ * TODO: Needs to be redesigned to solve performance issues.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)